lib/generators/godmin/resource/resource_generator.rb

```diff
@@ -2,12 +2,13 @@
 
 class Godmin::ResourceGenerator < Godmin::Generators::NamedBase
   argument :attributes, type: :array, default: [], banner: "attribute attribute"
-
+  class_option :"skip-navigation", :type => :boolean, :default => false, :description => "Skips extending navigation partial with new link"
```

Comment:
@telamon Could you remove the hash rockets as well?
`type: :boolean` and so on.